.env.sample

```diff
@@ -12,12 +12,18 @@ LISTEN_PORT=3000
 # is implemented.
 ALLOW_PRIVATE_ADDRESS=false
 REMOTE_ACTOR_FETCH_POSTS=10
+
+# File uploads and media storage:
 DRIVE_DISK=
-ASSET_URL_BASE=
-FS_ASSET_PATH=
-AWS_ACCESS_KEY_ID=
-AWS_SECRET_ACCESS_KEY=
-S3_REGION=
-S3_ENDPOINT_URL=
-S3_BUCKET=
-S3_FORCE_PATH_STYLE=false
+STORAGE_URL_BASE=
```

Comment:
If we had like a `WEB_DOMAIN` or `LOCAL_DOMAIN` setup, like mastodon, then we could in theory calculate a default for this based on that domain. If in production, force https, else, http.

That'd also give a better way to handle all the object IDs and such.

Reply:
I need to address this from Fedify first (which would require a new minor release).

Reply:
Yeah, definitely. For now, making `STORAGE_URL_BASE` mandatory works. We can relax that after you address this in Fedify.